README.md

```diff
@@ -151,6 +152,8 @@ You could use your browser to confirm this. If everything looks OK, you can shar
 
 ### Publish Your Change to Your Github Account
 
+Because our gh-pages branch here is without 'CNAME' file and its configuration has been changed, this gh-pages branch is not suitable for further pull request of merging if you only focus on modifying the content of this website. Instead, we choose to publish all of changes through the 'dev' branch and use the gh-pages branch only for checking those modifications. That's the philosophy here.
+
```

Comment:
To remove CNAME is an optional step. How about append one one statement like `If you removed CNAME and tweaked _config.yml by following the optional steps. Please note because our gh-pages branch here ...`

Beside, I think `Thats the philosophy here` is a redundant statement. May we remove it?